nearpy/tests/engine_tests.py

```diff
@@ -73,5 +73,69 @@ def test_retrieval_sparse(self):
             self.assertEqual(y_data, x_data)
             self.assertAlmostEqual(y_distance, 0.0, delta=delta)
 
+    def test_delete_vector(self):
+        # test case for 1 hash
+        dim = 5
+        unibucket = UniBucket('testHash')
+        engine = Engine(dim, lshashes=[unibucket])
+        # Index 20 random vectors (set their data to a unique string)
+        for index in range(20):
+            v = numpy.random.randn(dim)
+            engine.store_vector(v, index)
+        # Create random query vector
+        keys = set()
```

Comment:
You can just write
```python
bucket = engine.storage.buckets['testHash']['testHash']
keys = {i[1] for i in bucket}
```
I also suggest moving `'testHash'` into separate variable.